docs/en/source/installationGuide.rst

```diff
@@ -0,0 +1,213 @@
+*****************************************
+Installation Guide
+*****************************************
+
+In order to set-up the environment of Scorpio broker, the following dependency needs to be configured:-
+
+1. Eclipse.
+2. Server JDK.
+3. Apache Kafka.
+4. PostgreSQL
+
+
+Windows
+*****************************************
+
+Eclipse installation
+############################
+
+- **Download the Eclipse Installer.**:
+
+ Download Eclipse Installer from http://www.eclipse.org/downloads.Eclipse is hosted on many mirrors around the world. Please select the one closest to you and start to download the Installer.
+
+- **Start the Eclipse Installer executable**:
+
+ For Windows users, after the Eclipse Installer, the executable has finished downloading it should be available in your download directory. Start the Eclipse Installer executable. You may get a security warning to run this file. If the Eclipse Foundation is the Publisher, you are good to select Run.
+
+ For Mac and Linux users, you will still need to unzip the download to create the Installer. Start the Installer once it is available.
+
+- **Select the package to install**:
+
+ The new Eclipse Installer shows the packages available to Eclipse users. You can search for the package you want to install or scroll through the list. Select and click on the package you want to install.
+
+- **Select your installation folder**
+
+ Specify the folder where you want Eclipse to be installed. The default folder will be in your User directory. Select the ‘Install’ button to begin the installation.
+
+- **Launch Eclipse**
+
+ Once the installation is complete you can now launch Eclipse. The Eclipse Installer has done its work. Happy coding.
+
+
+JDK Setup
+##############
+
+- Start the JDK installation and hit the “Change destination folder” checkbox, then click 'Install.'
+
+**Note:-** Recommended version is JDK-11. Scorpio Broker is developed and tested with this version only.
+
+.. figure:: figures/jdk-1.png
+
+- Change the installation directory to any path without spaces in the folder name.
+
+
+After you've installed Java in Windows, you must set the  JAVA_HOME  environment variable to point to the Java installation directory.
+
+**Set the JAVA_HOME Variable**
+
+To set the JAVA_HOME variable:
+
+1. Find out where Java is installed. If you didn't change the path during installation, it will be something like this:
+
+ *C:\Program Files\Java\jdk1.version_detail*
+
+2. - In Windows 7 right-click **My Computer** and select **Properties** > **Advanced**.
+
+  OR
+
+ - In Windows 8 go to **Control Panel** > **System** > **Advanced System Settings**.
```

Comment:
/10

Reply:
I think we should put Windows 10 first, then 8 and finally 7

In Windows 10 go to **Control Panel** > **System and Security** > **System** >  **Advanced System Settings**.